### ISSUE TITLE
Redirect to /sites if logged in

### ISSUE THREAD

**Issue by dewey** [plausible/analytics#297]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the feature has not yet been requested.

## Feature request

Hey there, right now when I open https://plausible.io I see the landing page like a new user, then I click on "My Dashboard" in the top right corner. I think it would be clearer (and saving me a few clicks every day) by just redirecting logged in users to https://plausible.io/sites or just host the /sites on / if a user is logged in.

**Comment by ravivyas84** [comment]
+1 

I have had this thought too. 

A simpler solution would be redirecting sites.plausible.io to plausible.io/sites, so power users can start with "sites" in the address bar and the browser does the rest.

**Comment by ukutaht** [comment]
Agreed, unfortunately we sit behind a Netlify reverse proxy which doesn't support cookie-based redirects. When this feature goes live we will add a redirect rule: https://community.netlify.com/t/expanding-functionality-of-redirects/988/39
